lms/djangoapps/discussion/rest_api/views.py

```diff
@@ -109,6 +112,73 @@ def get(self, request, course_id):
         return Response(get_course(request, course_key))
 
 
+class CourseUsersView(DeveloperErrorViewMixin, APIView):
+    """
+    **Use Cases**
+
+        Fetch statistics about a user's activity in a course.
+
+    **Example Requests**:
+
+        GET /api/discussion/v1/courses/course-v1:ExampleX+Subject101+2015/activity_stats?order_by=activity
+
+    **Response Values**:
+
+
+    **Example Response**
+    ```json
+    {
+        "pagination": {
+            "count": 3,
+            "next": null,
+            "num_pages": 1,
+            "previous": null
+        },
+        "results": [
+            {
+                "active_flags": 3,
+                "inactive_flags": 0,
+                "replies": 13,
+                "responses": 21,
+                "threads": 32,
+                "username": "edx"
+            },
+            {
+                "active_flags": 1,
+                "inactive_flags": 0,
+                "replies": 6,
+                "responses": 8,
+                "threads": 13,
+                "username": "honor"
+            },
+            ...
+        ]
+    }
+    ```
+    """
+
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    permission_classes = (permissions.IsAuthenticated, permissions.IsAdminUser)
+
+    def get(self, request, course_key_string):
+        """Implements the GET method as described in the class docstring."""
+        form = UserCourseStatsForm(request.query_params)
```

Comment:
I don't understand this, what needs to be renamed?  Do you mean `form`? This is consistently called `form` across the codebase so I think we should go for consistency with the rest of the code here.

Reply:
sorry for the typo, yes I mean `form_query_string` I said that because it enhances the code readability. 
thanks.

Reply:
My point was that this variable has been called form in every other place, so I have named it form here for consistency. 

I have renamed it now, but IMO since it's the only form here, there isn't really any need to give it such a verbose name.